protocols/servlet/src/main/java/org/jboss/arquillian/protocol/servlet/v_3/ServletProtocolDeploymentPackager.java

```diff
@@ -91,7 +92,7 @@
     private Archive<?> handleArchive(JavaArchive applicationArchive, Collection<Archive<?>> auxiliaryArchives,
         JavaArchive protocol, Processor processor) {
         return handleArchive(
-            ShrinkWrap.create(WebArchive.class, "test.war")
+            ShrinkWrap.create(WebArchive.class, UUID.randomUUID().toString() + ".war")
```

Comment:
```suggestion
            ShrinkWrap.create(WebArchive.class, "test-" + UUID.randomUUID().toString() + ".war")
```

Same reason as above.